apps/parsley/src/components/LogRow/SubsectionHeader/index.tsx

```diff
@@ -1,40 +1,55 @@
-import styled from "@emotion/styled";
+import Icon from "@leafygreen-ui/icon";
 import { palette } from "@leafygreen-ui/palette";
 import { Body } from "@leafygreen-ui/typography";
 import { useLogWindowAnalytics } from "analytics";
 import { Row } from "components/LogRow/types";
-import { size } from "constants/tokens";
+import {
+  SectionHeaderWrapper,
+  SubsectionHeaderWrapper,
+} from "components/styles";
+import { SectionStatus } from "constants/logs";
 import { useLogContext } from "context/LogContext";
 import { CaretToggle } from "../CaretToggle";
 
 const { gray } = palette;
 
-interface SectionHeaderProps extends Row {
+interface SubsectionHeaderProps extends Row {
   commandName: string;
   functionID: string;
   commandID: string;
   open: boolean;
   step: string;
+  status: SectionStatus | undefined;
+  isTopLevelCommand: boolean;
 }
 
-const SubsectionHeader: React.FC<SectionHeaderProps> = ({
+const SubsectionHeader: React.FC<SubsectionHeaderProps> = ({
   commandID,
   commandName,
   functionID,
+  isTopLevelCommand,
   open,
+  status,
   step,
 }) => {
   const { sendEvent } = useLogWindowAnalytics();
   const { sectioning } = useLogContext();
+  const statusGlyph =
+    status === SectionStatus.Pass ? "CheckmarkWithCircle" : "XWithCircle";
+  const Wrapper = isTopLevelCommand
+    ? SectionHeaderWrapper
+    : SubsectionHeaderWrapper;
   return (
     <Wrapper aria-expanded={open} data-cy="section-header">
       <CaretToggle
         onClick={() => {
           sendEvent({
-            name: "Toggled section",
+            name: "Toggled section caret",
+            nested: !isTopLevelCommand,
             open: !open,
             "section.name": commandName,
             "section.type": "command",
+            status,
```

Comment:
I think the attributes should reference their parent object (otherwise we could ask, is it a task status? is it a build status? some other evergreen status?)
```suggestion
        "section.nested": !isTopLevelCommand,
        "section.open": !open,
        "section.status": status,
```

Reply:
I used "section.name" instead of "name" because "name" was already taken. The [docs](https://opentelemetry.io/docs/specs/semconv/general/attribute-naming/) mention that namespaces should be primiarily be used to avoid name collisions and not indicate hierarchies. The docs aren't really clear what is considered a naming collision though. Your example does kind of highlight a naming collision between status types.